docs/docs/kubernetes/scanning.md

```diff
@@ -0,0 +1,180 @@
+# Kubernetes
+
```

Comment:
Can we mention this is experimental here as below?
http://localhost:8000/trivy/docs/vulnerability/examples/filter/#by-open-policy-agent